tests/e2e/volume_health_test.go

```diff
@@ -3078,3 +3066,21 @@ var _ = ginkgo.Describe("Volume health check", func() {
 
 	})
 })
+
+func waitForVolumeHealthCheckEventToTriggerForPvc(ctx context.Context, client clientset.Interface,
+	volumeHealthAnnotation string, pvclaim *v1.PersistentVolumeClaim) (*v1.PersistentVolumeClaim, error) {
+	var pvc *v1.PersistentVolumeClaim
+	var err error
+	waitErr := wait.PollImmediate(pollTimeout, pollTimeoutSixMin, func() (bool, error) {
```

Comment:
Why are we using `pollTimeoutSixMin` we should use a value little larger than `healthStatusWaitTime`
We should also accept `healthStatusWaitTime` from env instead of keeping it at 2mins.